src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) {
+        FileReaderImpl fileReaderimpl = new FileReaderImpl();
+        List<String> lines = fileReaderimpl.read("reportToRead.csv");
+        System.out.println(lines);
+        System.out.println("Current working directory: " + System.getProperty("user.dir"));
```

Comment:
Test all services in the Main class. Use the resources directory for the files and push them. Move path to the constant fields